frontend/src/modules/login/components/LoginForm.tsx

```diff
@@ -21,67 +29,86 @@ export const LoginForm = () => {
-                {(fieldProps: FieldProps<string, LoginUserDTO>) => (
-                  <Input
-                    fieldProps={fieldProps}
-                    name='password'
-                    label='Password'
-                    type='password'
-                    id='password'
-                    borderColor='gray.300'
-                    bgColor='gray.50'
-                    color='gray.800'
-                  />
-                )}
-              </Field>
-            </Box>
-            <Box mb='4'>
-              <Button
-                data-cy='login-submit-btn'
-                formNoValidate
-                type='submit'
-                isLoading={mutation.isLoading}
-                width='full'
-                bgColor='gray.800'
-                color='gray.50'
-                _hover={{ bgColor: 'gray.800', color: 'gray.50' }}
-              >
-                Log In
-              </Button>
-            </Box>
-          </Form>
-        )}
-      </Formik>
-      <Flex justifyContent='right' color='gray.500'>
-        <NextLink href='/register' passHref>
-          <Link>Register</Link>
-        </NextLink>
-      </Flex>
-    </Box>
+    <Formik
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+      onSubmit={onSubmit}
+    >
+      {() => (
+        <Form noValidate>
+          <FormLayout>
+            <Field name='email' type='email'>
+              {(fieldProps: FieldProps<string, LoginUserDTO>) => (
+                <Input
+                  fieldProps={fieldProps}
+                  name='email'
+                  label='Email'
+                  type='email'
+                  id='email'
+                />
+              )}
+            </Field>
+            <Flex direction='column'>
+              <FormControl>
+                <FormLabel
+                  htmlFor='password'
+                  aria-labelledby='password'
+                  fontWeight='semibold'
+                >
+                  Password
+                </FormLabel>
+                <Field name='password' type='password'>
+                  {(fieldProps: FieldProps<string, ChangePasswordDTO>) => (
+                    <InputGroup>
+                      {fieldProps.field.value?.length > 0 ? (
+                        <InputRightElement
+                          alignItems='center'
+                          justifyContent='center'
+                          h={10}
+                          px='4'
+                        >
+                          <IconButton
+                            aria-label='Show password'
+                            size='sm'
+                            variant='ghost'
+                            icon={showPassword ? <FaEye /> : <FaEyeSlash />}
+                            onClick={() => setShowPassword(!showPassword)}
+                          />
+                        </InputRightElement>
+                      ) : null}
```

Comment:
Additional e2e tests can be implemented here as we now have show/hide for the Login's password field